### ISSUE TITLE
network: failed to start networkd on system boot with "Could not enumerate links: Exchange full"

### ISSUE THREAD

**Issue by yuwata** [systemd/systemd#16319]
With current git master,
```
Jun 29 16:27:54 systemd[1]: Starting Network Service...
Jun 29 16:27:55 systemd-networkd[581]: SELinux enabled state cached to: disabled
Jun 29 16:27:55 systemd-networkd[581]: sd-device-monitor: No permission to stat PID1's netns, unable to determine if we are in host netns: Permission denied
Jun 29 16:27:55 systemd-networkd[581]: Bus bus-api-network: changing state UNSET → OPENING
Jun 29 16:27:55 systemd-networkd[581]: Bus bus-api-network: changing state OPENING → AUTHENTICATING
Jun 29 16:27:56 systemd-networkd[581]: Registering bus object implementation for path=/org/freedesktop/LogControl1 iface=org.freedesktop.LogControl1
Jun 29 16:27:56 systemd-networkd[581]: timestamp of '/etc/systemd/network' changed
Jun 29 16:27:56 systemd-networkd[581]: No virtualization found in DMI
Jun 29 16:27:56 systemd-networkd[581]: No virtualization found in CPUID
Jun 29 16:27:56 systemd-networkd[581]: Virtualization XEN not found, /proc/xen does not exist
Jun 29 16:27:56 systemd-networkd[581]: This platform does not support /proc/device-tree
Jun 29 16:27:56 systemd-networkd[581]: UML virtualization not found in /proc/cpuinfo.
Jun 29 16:27:56 systemd-networkd[581]: This platform does not support /proc/sysinfo
Jun 29 16:27:56 systemd-networkd[581]: Found VM virtualization none
Jun 29 16:27:56 systemd-networkd[581]: /usr/lib/systemd/network/80-container-host0.network: Conditions in the file do not match the system environment, skipping.
Jun 29 16:27:56 systemd-networkd[581]: Could not enumerate links: Exchange full
Jun 29 16:27:56 systemd-networkd[581]: Bus bus-api-network: changing state AUTHENTICATING → HELLO
Jun 29 16:27:56 systemd-networkd[581]: Sent message type=method_call sender=n/a destination=org.freedesktop.DBus path=/org/freedesktop/DBus interface=org.freedesktop.DBus member=Hello cookie=1 reply_cookie=0 signature=n/a error-name=n/a >
Jun 29 16:27:56 systemd-networkd[581]: Sent message type=method_call sender=n/a destination=org.freedesktop.DBus path=/org/freedesktop/DBus interface=org.freedesktop.DBus member=RequestName cookie=2 reply_cookie=0 signature=su error-name>
Jun 29 16:27:56 systemd-networkd[581]: Sent message type=method_call sender=n/a destination=org.freedesktop.DBus path=/org/freedesktop/DBus interface=org.freedesktop.DBus member=AddMatch cookie=3 reply_cookie=0 signature=s error-name=n/a>
Jun 29 16:27:56 systemd-networkd[581]: Got message type=method_return sender=org.freedesktop.DBus destination=:1.12 path=n/a interface=n/a member=n/a cookie=4294967295 reply_cookie=1 signature=s error-name=n/a error-message=n/a
Jun 29 16:27:56 systemd-networkd[581]: Bus bus-api-network: changing state HELLO → RUNNING
Jun 29 16:27:56 systemd-networkd[581]: Bus bus-api-network: changing state RUNNING → CLOSED
Jun 29 16:27:56 systemd[1]: systemd-networkd.service: Main process exited, code=exited, status=1/FAILURE
Jun 29 16:27:56 systemd[1]: systemd-networkd.service: Failed with result 'exit-code'.
Jun 29 16:27:56 systemd[1]: Failed to start Network Service.
Jun 29 16:27:56 systemd[1]: systemd-networkd.service: Consumed 2.294s CPU time.
```
However, after login the system, by manually restart networkd, it works well.

**Comment by yuwata** [comment]
This is caused by 3691bcf3c5eebdcca5b4f1c51c745441c57a6cd1.

**Comment by yuwata** [comment]
Ah, this may be already fixed by #15942. I use systemd-245.6-2.fc32.x86_64 and only networkd is built from current git master.
